### PR TITLE
chore: remove unused dependencies

### PR DESCRIPTION
**Pull request by ChristianMurphy**
##### Checklist

- [x] the [contributor license agreement][] is signed
- [x] commit message follows [commit guidelines][]
- [ ] tests are included
- [ ] screenshots are included showing significant UI changes
- [ ] documentation is changed or added

##### Description of change

closes #2187 

[contributor license agreement]: https://www.apereo.org/node/676
[commit guidelines]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

File: Source/Plugins/Core/com.equella.core/js/package.json [hunk 1]
```diff
@@ -99,15 +99,11 @@
     "@types/sprintf-js": "1.1.2",
     "@types/tinymce": "4.5.24",
     "@types/uuid": "8.3.0",
-    "argparse": "1.0.10",
     "babel-jest": "26.3.0",
     "babel-loader": "8.1.0",
-    "combined-stream2": "1.1.2",
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/package.json:105]: this appears to be a transitive dependency

File: Source/Plugins/Core/com.equella.core/js/package.json [hunk 2]
```diff
@@ -99,15 +99,11 @@
...
     "core-js": "3.6.5",
     "cross-env": "7.0.2",
     "cross-spawn": "7.0.3",
-    "css-vars-ponyfill": "2.3.2",
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/package.json:109]: this appears to be a transitive dependency

File: Source/Plugins/Core/com.equella.core/js/package.json [hunk 3]
```diff
@@ -99,15 +99,11 @@
...
-    "envify": "4.1.0",
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/package.json:110]: this appears to be a transitive dependency

File: Source/Plugins/Core/com.equella.core/js/package.json [hunk 4]
```diff
@@ -99,15 +99,11 @@
     "@types/sprintf-js": "1.1.2",
     "@types/tinymce": "4.5.24",
     "@types/uuid": "8.3.0",
-    "argparse": "1.0.10",
```
Comment (ChristianMurphy) [inline Source/Plugins/Core/com.equella.core/js/package.json:102]: this appears to be a transitive dependency

**Review by edalex-ian — APPROVED** [review]
Good stuff. Thank you. :+1:

**Pull request merged**